octue/resources/child.py

```diff
@@ -69,12 +68,15 @@ def ask(
         push_endpoint=None,
         asynchronous=False,
         retry_count=0,
+        cpus=None,
+        memory=None,
+        ephemeral_storage=None,
         raise_errors=True,
         max_retries=0,
         prevent_retries_when=None,
         log_errors=True,
         timeout=86400,
-        maximum_heartbeat_interval=300,
+        maximum_heartbeat_interval=360,
```

Comment:
I've seen this repeated throughout, why not have a MAXIMUM_HEARTBEAT_INTERVAL parameter?